env/WCOSS2.env

```diff
@@ -223,7 +223,7 @@ elif [[ "${step}" = "postsnd" ]]; then
     export OMP_NUM_THREADS=1
 
     export NTHREADS_POSTSND=${NTHREADS1}
-    export APRUN_POSTSND="${APRUN} --depth=${NTHREADS_POSTSND} --cpu-bind depth"
+    export mpmd_opt="-ppn 21 ${mpmd_opt}"
```

Comment:
@BoCui-NOAA @WalterKolczynski-NOAA Was it intentional to get rid of `APRUN_POSTSND`?  This is referenced by `ush/gfs_bufr_netcdf.sh`.

Reply:
Yes, the 'APRUN_POSTSND' setting is no longer needed, and the ush/gfs_bufr_netcdf.sh  script is no longer in use.